code/game/objects/items/stacks/medical.dm

```diff
@@ -104,7 +104,27 @@
 			if(stop_bleeding)
 				if(!H.bleedsuppress) //so you can't stack bleed suppression
 					H.suppress_bloodloss(stop_bleeding)
+

```

Comment:
You should look into making both of these into a single proc so that you aren't duplicating similar code